src/Kiota.Builder/OpenApiDocumentDownloadService.cs

```diff
@@ -18,7 +18,7 @@
 using Microsoft.OpenApi.Validations;
 
 namespace Kiota.Builder;
-internal class OpenApiDocumentDownloadService
+public class OpenApiDocumentDownloadService
```

Comment:
Why has the access modifier changed here?

Reply:
That class is in the same assembly (`Kiota.Builder`) so `internal` should still work for it

Reply:
It doesn't work as internal, so it has to be changed to public, unless without major refactor, since it is used as a parameter in a public method